src/reducers/auth.lua

```diff
@@ -3,13 +3,14 @@ local e = _G.PolarisNav
 local save = e.op.persistent_save
 local load = e.op.persistent_load
 
-local function set_auth(uid, tok, ses, code, att)
+local function set_auth(uid, tok, ses, code, att, guest)
 	local auth = {
 		UserId = tonumber(uid);
 		session = ses;
 		token = tok;
 		Code = code;
 		attempts = att;
+        guestMode = guest;
```

Comment:
Indentation. In the future, boolean variables like this often start with an "is_" prefix for readability. It's okay as is here though.